laravel/resources/views/layouts/footer-main.blade.php

```diff
@@ -0,0 +1,21 @@
+<div class="container mt-auto">
+    <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
+        <div class="col-md-12 d-flex align-items-center justify-content-between">
+            <span class="text-muted">
+                &copy; {{\Illuminate\Support\Carbon::now()->format('Y')}} by
```

Comment:
[SUGGESTION]

Should also work and make it more readable:
```suggestion
                &copy; {{ Carbon::now()->format('Y') }} by
```